grittyDV/tka-website
# Project Setup with Firebase Integration

This PR sets up the initial project structure with Firebase integration:

1. Project Structure:
   - Organized components into `/src/components/`
   - Created pages in `/src/pages/`
   - Clean configuration setup

2. Features Added:
   - Firebase initialization and analytics
   - React Router setup
   - Tailwind CSS configuration
   - Webpack development server

3. Components:
   - Header with navigation
   - Footer with site links
   - Layout wrapper
   - Basic page components

4. Configuration:
   - Webpack setup for development and production
   - PostCSS configuration for Tailwind
   - Clean package.json with necessary dependencies

To test:
1. `npm install`
2. `npm start`

The site should now have working navigation and Firebase integration.
